src/scores/categorical/contingency_impl.py

```diff
@@ -405,25 +406,27 @@ def critical_success_index(self) -> xr.DataArray:
 
     def peirce_skill_score(self) -> xr.DataArray:
         """
-        Identical to Hanssen and Kuipers discriminant and the true skill statistic
+        Identical to Hanssen and Kuipers discriminant and the true skill statistic.
 
         How well did the forecast separate the "yes" events from the "no" events?
 
         Returns:
-            An xarray object containing the Peirce Skill Score
+            xr.DataArray: An xarray object containing the Peirce Skill Score
 
         .. math::
             \\text{Peirce skill score} = \\frac{\\text{true positives}}{\\text{true positives} + \\text{false negatives}} - \\frac{\\text{false positives}}{\\text{false positives} + \\text{true negatives}}
 
         Notes:
             - Range: -1 to 1, 0 indicates no skill. Perfect score: 1.
-            - "True positives" is the same as "hits"
-            - "False negatives" is the same as "misses"
-            - "False positives" is the same as "false alarms"
-            - "True negatives" is the same as "correct negatives"
+            - "True positives" is the same as "hits".
+            - "False negatives" is the same as "misses".
+            - "False positives" is the same as "false alarms".
+            - "True negatives" is the same as "correct negatives".
 
         References:
-            https://www.cawcr.gov.au/projects/verification/#HK
+            - https://www.cawcr.gov.au/projects/verification/#HK
+            - Peirce, C.S., 1884. The numerical measure of the success of predictions. \
+              Science, 4(93), pp.453-454. https://doi.org/10.1126/science.ns-4.93.453.b
```

Comment:
```suggestion
              Science, ns-4(93), pp.453-454. https://doi.org/10.1126/science.ns-4.93.453.b
```
Science website says "Vol ns-4, Issue 93" and seem to want it cited as "ns-4" and not "4"